demo/src/screens/listScreens/ConversationListScreen.js

```diff
@@ -1,113 +1,197 @@
-import React, {Component} from 'react';
```

Comment:
The performance is much better, but I think I notice a bug. 
open the drawer of one of the items, then close its drawer, then try to open the same item's drawer again. it won't open, it keeps getting closed.

Reply:
Fixed. Thanks!